sgp40_voc_index/sgp40_voc_index_example_usage.c

```diff
@@ -54,7 +54,7 @@ int main(void) {
     /* Run one measurement per second, persist state every hour */
     while (1) {
         err = sensirion_measure_voc_index_with_rh_t(
-            &voc_index, &temperature_celsius, &relative_humidity_percent);
+            &voc_index, &relative_humidity_percent, &temperature);
```

Comment:
```suggestion
            &voc_index, &relative_humidity_percent, &temperature_celsius);
```

Reply:
This is kind of embarrassing :laughing: I blame autocomplete for it :stuck_out_tongue_winking_eye: